src/deobfuscator/inlineDecodedStrings.ts

```diff
@@ -11,18 +11,17 @@ export default {
   visitor: options => ({
     CallExpression(path) {
       options?.decoders.forEach(decoder => {
-        const matcher = m.callExpression(
-          m.identifier(decoder.name),
-          m.anything()
-        );
+        const matcher = m.callExpression(m.identifier(decoder.name), m.anything());
 
         if (matcher.match(path.node) && t.isLiteral(path.node.arguments[0])) {
           const args = path.node.arguments.map(arg => {
             if (t.isNumericLiteral(arg) || t.isStringLiteral(arg)) {
               return arg.value;
             }
           });
-          path.replaceWith(t.stringLiteral(options.vm.decode(decoder, args)));
+
+          let decoded: string | undefined = options.vm.decode(decoder, args);
+          path.replaceWith(decoded ? t.stringLiteral(decoded) : t.tsUndefinedKeyword());
```

Comment:
I think tsUndefinedKeyword is for types only, not a JS Expression. Correct one would be identifier:
https://astexplorer.net/#/gist/30b928673f55813768923e8643300a88/f444dc787bb4fbd12c244f0f008b2fc7687b37c1

Reply:
Would be better, so in the future other transforms don't have to check for 2 kinds of undefined